sheet_to_triples/run.py

```diff
@@ -42,11 +43,12 @@ def __init__(
             self.graph = rdf.graph_from_model(model)
         else:
             self.graph = rdf.graph_from_triples(())
-        self.non_unique = non_unique
-        if self.non_unique is None:
-            self.non_unique = _default_non_unique
-        self.resolve_same = resolve_same
-        self.drop_duplicates = drop_duplicates
+        self.normalisation_params = {
+            'non_uniques': _default_non_unique if non_unique is None else non_unique,
```

Comment:
Probably a copy of default dict, because you manipulate it at [line 73](https://github.com/VisualMeaning/sheet-to-triples/pull/140/files#diff-e5049948f65fa38ae06751686f49a0c25b2e1c643712c7c0a38132c081e252edR73).

Reply:
Yeah, it's not 100% necessary because we only ever have one Runner at a time, but still good practice.